### PR TITLE
Unsupported token queries

### LINKED ISSUE #6397: TokenGetAccountNftInfosHandler
Permanently `NOT_SUPPORTED`.

### LINKED ISSUE #6400: TokenGetNftInfosHandler
Permanently `NOT_SUPPORTED`.

### PR DESCRIPTION
**Pull request by kimbor**
Two token queries which are not supported

**Related issue(s)**:

Fixes #6397
Fixes #6400

### CODE REVIEW

File: hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java [hunk 1]
```diff
@@ -45,7 +49,7 @@ public TokenGetNftInfoHandler() {
     @Override
     public QueryHeader extractHeader(@NonNull final Query query) {
         requireNonNull(query);
-        return query.tokenGetInfoOrThrow().header();
+        return query.tokenGetNftInfoOrThrow().header();
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java:52]: Seems wrong file is modified . It should be `TokenGetNftInfosHandler`

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGetAccountNftInfoHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.token.impl.test.handlers;
+
+import static com.hedera.hapi.node.base.ResponseCodeEnum.NOT_SUPPORTED;
+import static com.hedera.node.app.spi.fixtures.workflows.ExceptionConditions.responseCode;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGetAccountNftInfoHandlerTest.java:20]: Wrong test class modified `TokenGetAccountNftInfosHandlerTest.java`

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGetNftInfoHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.token.impl.test.handlers;
+
+import static com.hedera.hapi.node.base.ResponseCodeEnum.NOT_SUPPORTED;
+import static com.hedera.node.app.spi.fixtures.workflows.ExceptionConditions.responseCode;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenGetNftInfoHandlerTest.java:21]: same here , should be `TokenGetNftInfosHandlerTest.java`

**Review by Neeharika-Sompalli — CHANGES_REQUESTED** [review]

File: hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java (hunk 1 above)
Comment (povolev15) [reply hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java:52]: I will fix it

Comment (povolev15) [reply hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java:48]: that specific change is correct by the way no, @Neeharika-Sompalli. It is supposed to be `tokenGetNftInfoOrThrow()`

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM ! Thanks @kimbor @povolev15

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]

**Pull request merged**